### PR TITLE
translating policies and telemetry overview into Chinese

### PR DESCRIPTION
**Pull request by rootsongjc**
@gyliu513 Already reviewed.

### CODE REVIEW

File: content_zh/docs/concepts/policies-and-telemetry/overview.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Istio 提供灵活的模型来执行授权策略，并为网格中的服务收集遥测数据。
+
+基础设施后端旨在提供用于构建服务的支持功能。它们包括访问控制系统，遥测捕获系统，配额执行系统，计费系统等等。服务传统上直接与这些后端系统集成，创建硬耦合，还有沾染特定的语义和使用选项。
+
+Istio 提供统一抽象，使得 Istio 可以与一组开放式基础设施后端进行交互。这样做是为了给运维提供丰富而深入的控制，同时不给服务开发人员带来负担。Istio 旨在改变层与层之间的边界，以减少系统复杂性，消除服务代码中的策略逻辑并将控制权交给运维。
+
+Mixer 是负责提供策略控制和遥测收集的 Istio 组件：
+
+{{< image width="75%" ratio="49.26%"
+    link="../img/topology-without-cache.svg"
```
Comment (gyliu513) [inline content_zh/docs/concepts/policies-and-telemetry/overview.md]: Can the image be found?

File: content_zh/docs/concepts/traffic-management/load-balancing.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ keywords: [traffic-management,load-balancing]
     caption="发现与负载均衡"
     >}}
 
-如上图所示，网格中的服务使用其 DNS 名称访问彼此。服务的所有 HTTP 流量都会通过 Envoy 自动重新路由。Envoy 在负载均衡池中的实例之间分发流量。虽然 Envoy 支持多种[复杂的负载均衡算法][https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing]，但 Istio 目前仅允许三种负载平衡模式：轮循、随机和带权重的最少请求。
+如上图所示，网格中的服务使用其 DNS 名称访问彼此。服务的所有 HTTP 流量都会通过 Envoy 自动重新路由。Envoy 在负载均衡池中的实例之间分发流量。虽然 Envoy 支持多种[复杂的负载均衡算法](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/load_balancing)，但 Istio 目前仅允许三种负载平衡模式：轮循、随机和带权重的最少请求。
```
Comment (gyliu513) [inline content_zh/docs/concepts/traffic-management/load-balancing.md]: For future PR, it is better to create such PR separately for better tracing.

File: content_zh/docs/concepts/policies-and-telemetry/overview.md (hunk 1 above)
Comment (rootsongjc) [reply content_zh/docs/concepts/policies-and-telemetry/overview.md]: Yes, there's no problem.

File: content_zh/docs/concepts/traffic-management/load-balancing.md (hunk 2 above)
Comment (rootsongjc) [reply content_zh/docs/concepts/traffic-management/load-balancing.md]: OK, I will send PR for files one by one.

**Pull request merged**